### PR TITLE
Fixes borgs radio help message

### PR DESCRIPTION
**Pull request by AnturK**
Give me suggestion what AI one should say.

### CODE REVIEW

File: code/modules/jobs/job_types/job.dm [hunk 1]
```diff
@@ -147,6 +147,8 @@
 /datum/job/proc/map_check()
 	return TRUE
 
+/datum/job/proc/radio_help_message(mob/M)
+	to_chat(M, "<b>To speak on your departments radio, use the :h button. To see others, look closely at your headset.</b>")
```
Comment (MrDoomBringer) [inline code/modules/jobs/job_types/job.dm]: There isn't a ":h" button. Should say something like: _prefix your message with ":h:" to speak on your department's radio_

**Review by MrDoomBringer — CHANGES_REQUESTED** [review]

File: code/modules/jobs/job_types/job.dm (hunk 1 above)
Comment (AnturK) [reply code/modules/jobs/job_types/job.dm]: Yeah, changed wording a bit here.

**Comment by yoyobatty** [comment]
Standardized and neat lgtm

File: code/modules/jobs/job_types/job.dm [hunk 2]
```diff
@@ -147,6 +147,8 @@
 /datum/job/proc/map_check()
 	return TRUE
 
+/datum/job/proc/radio_help_message(mob/M)
+	to_chat(M, "<b>Prefix your message with :h to speak on your departments radio. To see others, look closely at your headset.</b>")
```
Comment (SpaceManiac) [inline code/modules/jobs/job_types/job.dm]: should be "department's"

**Pull request merged**